### ISSUE TITLE
Error on startup while the bot is in a big server or upon joining one.

### ISSUE THREAD

**Issue by Totti56** [SideProjectGuys/invite-manager-bot#204]
Whenever the bot starts up while being in this big server or upon joining this big server it shows this huge list of the same error. After the error the bot just proceeds. 
However, I think that this error is the cause of certain data missing in the database, seen whenever I do !leaderboard and obvious large invites aren't being shown.
![image](https://user-images.githubusercontent.com/5383805/116575274-c7b38780-a90e-11eb-97e3-d98e2caee6e9.png)
So, my question is if there is any way to fix this problem? Thank you.

Oh and also, there was this one error where stage channels weren't recognized, not sure if this error can be fatal.

**Comment by Totti56** [comment]
It seems like I managed to fix this error. 
However, it seems like this error was unrelated to the issue where when I use !leaderboard it still shows missing invites at the top invites, while they do exist in the database table. Any clue on why !leaderboard has missing stats?

**Comment by chaun14** [comment]
I've also randomly the leaderboard missing entries issue. It seems like the leaderboard cache isn't working correctly when there are a large amount of data.

**Comment by Totti56** [comment]
> I've also randomly the leaderboard missing entries issue. It seems like the leaderboard cache isn't working correctly when there are a large amount of data.

Gotcha! So, there is not really a way to fix this problem? :(

Also, I managed to fix the error from the top by updating the discord js wrapper Eris, because we have stage channels in our server which are new. However, by updating the package there are a few errors caused by that or warnings where the script does not recognize certain these old function names etc. idk. But yeah, my main issue I have at the moment is the leaderboard cache.

**Comment by chaun14** [comment]
I've never really searched to fix this problem.

You can find the project with updated eris version in some forks.

PS: Eris is not a discord.js wrapper

**Comment by Totti56** [comment]
> I've never really searched to fix this problem.
> 
> You can find the project with updated eris version in some forks.
> 
> PS: Eris is not a discord.js wrapper

Yes thank you, oops sorry I'm not really an expert in stuff like this lol, thank you for correcting me :D
